smithy-model/src/test/resources/software/amazon/smithy/model/knowledge/member-property-index.smithy

```diff
@@ -1,12 +1,5 @@
 $version: "2.0"
 
-metadata suppressions = [
```

Comment:
This shouldn't have been there and is one of the reasons I missed this in the first PR :(. Removing now to avoid any confusion in the future.